src/vs/workbench/parts/extensions/electron-browser/extensionEditor.ts

```diff
@@ -369,6 +392,22 @@ export class ExtensionEditor extends BaseEditor {
 		this.extensionActionBar.push([disabledStatusAction, reloadAction, updateAction, enableAction, disableAction, installAction, maliciousStatusAction], { icon: true, label: true });
 		this.transientDisposables.push(enableAction, updateAction, reloadAction, disableAction, installAction, maliciousStatusAction, disabledStatusAction);
 
+		const ignoreAction = this.instantiationService.createInstance(IgnoreAction);
+		ignoreAction.extension = extension;
+		ignoreAction.onIgnored = () => {
```

Comment:
Maybe its just me, but it feels a bit weird to "set" the `onIgnored` callback like this.
How about passing this ignoreCallback in the constructor for the action?

Thoughts?